examples/local_ratelimit/ratelimit-envoy.yaml

```diff
@@ -95,6 +99,21 @@ static_resources:
                     key: x-local-rate-limit
                     value: 'true'
               local_rate_limit_per_downstream_connection: false
+              descriptors:
+              - entries:
+                - key: generic_key
+                  value: test1
+                token_bucket:
+                  max_tokens: 3
+                  tokens_per_fill: 3
+                  fill_interval: 6s
+              - entries:
```

Comment:
this doesnt appear to be used in either the docs or the verification - is it necessary ?

Reply:
it is the specific rate limit of descriptor `test1`, the route of upstream service in listener 10000 will match descriptor `test1`, as a result, this route can be requested 3 times at most in 6s, at the same time, the default rate limit will not be applied.